src/server.js

```diff
@@ -0,0 +1,10 @@
+const http = require('http');
+
+const port = process.argv.PORT || 5000;
+
+const server = http.createServer();
```

Comment:
call for router function here and require it at the top of the server.js file just after the http